de.json

```diff
@@ -100,10 +100,10 @@
             "languagePicker" : {
                 "trigger" : "Sprache",
                 "title" : "Sprache auswählen",
-                "currentLanguage" : "Du verwendest derzeit [language]",
+                "currentLanguage" : "Du verwendest derzeit die Sprache [language]",
                 "error" : "Fehler beim Herunterladen der Sprachdatei",
-                "authors" : "Übersetzt von:",
-                "cta" : "Klicke hier, um zu [language] zu wechseln",
+                "authors" : "Übersetzt von: derju0 [info@eternal5.de]",

```

Comment:
I believe this phrase isn't actually used n the website, but anyway: The name of authors would be appended automatically. So you shouldn't add your name here.